src/ruby/puppet-server-lib/puppet/server/puppet_config.rb

```diff
@@ -22,7 +22,7 @@ def self.initialize_puppet(puppet_config)
     Puppet[:trace] = true
 
     # Crank Puppet's log level all the way up and just control it via logback.
-    Puppet::Util::Log.level = :debug
+    Puppet[:log_level] = "debug"
```

Comment:
I think this worked before because `log_level` wasn't a setting -- so you could set the global `level` here, and puppet would never overwrite it.

I think it stopped working, because as soon as we initialize settings below, it reads the default `log_level` setting of `notice` and resets the level from `debug` to `notice`. I *think* you could move the line you had previously after settings initialization, but then I think you'd lose debug messages that occur *while* settings are initialized.

So :+1: